poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.0 and should not be changed by hand.
 
```

Comment:
Not sure whether the version of gitpython has been updated in the poetry.lock?

Reply:
https://github.com/NHSDigital/communications-manager-api/blob/69cc9185e93c55c2eeb8b2c1e7809d6a053a6105/poetry.lock#L537 Seems this was already updated in main?! (will resolve comment)